e2e/interchaintestv8/chainconfig/chain_config.go

```diff
@@ -19,7 +19,7 @@ func IbcGoChainSpec(name, chainId string) *interchaintest.ChainSpec {
 			Images: []ibc.DockerImage{
 				{
 					Repository: "ghcr.io/cosmos/ibc-go-wasm-simd", // FOR LOCAL IMAGE USE: Docker Image Name
-					Version:    "main",                            // FOR LOCAL IMAGE USE: Docker Image Tag
+					Version:    "gjermund-revert-to-0.50",         // FOR LOCAL IMAGE USE: Docker Image Tag
```

Comment:
Will revert to main after https://github.com/cosmos/ibc-go/pull/7932 has been merged